proto/proto-backplane-grpc/src/main/proto/build-cpp-protos.sh

```diff
@@ -2,11 +2,11 @@
 
 set -euxo pipefail
 
-: ${DHCPP_LOCAL:=$HOME/dhcpp/local}
+: ${DHCPP_LOCAL:=$HOME/dhcpp}
 : ${CPP_PROTO_BUILD_DIR:=build}
 
 mkdir -p "${CPP_PROTO_BUILD_DIR}"
-$DHCPP_LOCAL/protobuf/bin/protoc \
+$DHCPP_LOCAL/bin/protoc \
```

Comment:
I kind of wonder two things:
1. whether this should be DHCPP not DHCPP_LOCAL  (i.e. why "local")
2. whether this should not be set at all, and we should tell people to use the same $DHCPP variable they got when they ran the "env.sh" file we generated for them

Reply:
I think you are right about 1, changed and pushed.
Ref 2, there is one level of indirection that is always necessary, something needs to point to where the env.sh file lives; so we need something to know where the $PREFIX deployment was made.  The way this code is written, it is essentially assuming a default (which I don't think is most of the time used... this script is meant for manual execution, and you/me should come here with the env.sh file sourced already).  The syntax
```
: ${POPO:=popo_default}
```
essentially tells the shell that if the `POPO` environment variable is already defined, use its value, otherwise if it is not defines, use `popo_default`.
So maybe I would question the default and we could change the logic to "check if the variable is not defined, and if it is not, bail with an error".
?

Reply:
We agreed on a check of the var + fail instead of default.